addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/internal/SeneyeService.java

```diff
@@ -153,34 +154,36 @@ private String GetData(String request) {
 
         String url = "https://api.seneye.com/v1/devices" + request;
 
-        HttpClient getClient = new HttpClient();
-
-        GetMethod getMethod = new GetMethod(url);
-        getMethod.getParams().setParameter(HttpMethodParams.RETRY_HANDLER, new DefaultHttpMethodRetryHandler(3, false));
-        getMethod.getParams().setParameter(HttpMethodParams.USER_AGENT,
-                "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
-        getMethod.addRequestHeader("Accept", "application/json");
+        Request getMethod = httpClient.newRequest(url);
+        // getMethod.getParams().setParameter(HttpMethodParams.RETRY_HANDLER, new DefaultHttpMethodRetryHandler(3,
+        // false));
+        getMethod.agent("Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
```

Comment:
Do you get blacklisted or something if you do not fake the useragent here?

Reply:
ok, removed